pystiche_papers/ulyanov_et_al_2016/_nst.py

```diff
@@ -23,6 +24,26 @@
 __all__ = ["training", "stylization"]
 
 
+class OptionalResizeCenterCropToMultiple(nn.Module):
```

Comment:
Shouldn't this be in the `_data` module? I mean we have 

https://github.com/pystiche/papers/blob/2c3972838cb9b8998c86cd30fd549bb94b30d532/pystiche_papers/ulyanov_et_al_2016/_data.py#L84

Should we maybe rename that to `training_content_transform` and have a `stylization_content_transform`? Or maybe we don't need to rename the old one and just call this one `stylization_transform` since at this stage there is no longer a content or style image.